source/documentation/runbooks/010-disaster-recovery-plan.html.md.erb

```diff
@@ -0,0 +1,129 @@
+
+## Scope
+- **Application**: Django-based Python application
+- **Backend services**: DataHub
+- **Hosting**: MoJ's Cloud Platform: AWS EKS
+- **Database**: AWS RDS PostgreSQL
+- **Key Components**:
+  - Django application code
+  - DataHub application
+  - AWS RDS PostgreSQL database
+  - Docker images stored in Amazon ECR
+
+## Business appetite for time to recovery
+- Our service has not yet reached full deployment within the organisation, and there are no services dependant upon its function. As such, the service could become unresponsive for extended periods without compromising end-user deliver objectives.
+- We aim to be able to restore all service within 48 hours of disruption.
+
+## Disaster Scenarios
+
+| Disaster                                                                                                           | Severity | Likelihood | Recovery/Mitigation                                                                                                                                                                                                                       |
+|--------------------------------------------------------------------------------------------------------------------|----------|------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Disruption to hosting services                                                                                     | High     | Low        | - Following recovery of the platform, we may need to [re-deploy our applications](#issues-at-cloud-platform-level) |
+| AWS Region outage                                                                                                  | High     | Low        | - Wait<br>- This may lead to Disruption to hosting services (above) |
+| Loss of data due to misconfiguration of AWS resources in cloud-platform-environments repo                          | Medium   | Low        | - [Restore database from RDS snapshot](#database-recovery) |
+| Intentional disruption by malicious actor e.g. Distributed Denial of Service (DDoS)                                | High     | Medium     | - [Horizontal auto-scaling] with sensible scaling ceiling<br>- Rate limiting on NGINX ingress<br>- [AWS Shield Standard] (managed DDoS protection service) safeguards applications running on AWS. CP are looking into rolling out [AWS Shield Advanced] for more comprehensive integrated protection |
+| Deletion / corruption of application database data                                                                 | Medium   | Low        | - [Restore from snapshot](#database-recovery) |
```

Comment:
I think there are a few possibilities here depending on the data in question

- Currently, any data loss from the Find MoJ data database is acceptable and we could choose not to restore
- If data is not completely deleted, recovery via a database migration will probably be quicker than a full restore
- If data is unintentionally deleted from datahub (e.g. by running the wrong CLI command), it should be recoverable by rerunning the scheduled ingestion from github actions
- As a last resort, we can restore from snapshot

Reply:
Done